### PR TITLE
[doc][troubleshooting] add wrong cluster problem

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 1]
```diff
@@ -47,9 +47,13 @@ nebula> GET CONFIGS graph:load_data_interval_secs
 
 ## 使用 docker 启动后，执行命令时报错
```
Comment (wilsonyou) [inline docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md]: Docker

File: docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 2]
```diff
@@ -47,9 +47,13 @@ nebula> GET CONFIGS graph:load_data_interval_secs
...
 
-可能的原因是 docker 的 IP 地址和默认配置中的监听地址不一致(默认是 172.17.0.2)，因此这里需要修改默认配置中的监听地址.
+可能的原因是 docker 的 IP 地址和默认配置中的监听地址不一致(默认是 172.17.0.2)，因此这里需要修改默认配置中的监听地址。
 
 1. 首先在容器中执行 ifconfig 命令，查看您的容器地址，这里假设您的容器地址是172.17.0.3，那么就意味着您需要修改默认配置的IP地址。
```
Comment (wilsonyou) [inline docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md]: `ifconfig`

File: docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 3]
```diff
@@ -47,9 +47,13 @@ nebula> GET CONFIGS graph:load_data_interval_secs
...
 2. 然后进入配置目录(cd /usr/local/nebula/etc), 查找所有IP地址配置的位置(grep "172.17.0.2" . -r)。
 3. 修改上一步查到的所有IP地址为您的容器地址(172.17.0.3)。
 4. 最后重新启动所有服务。
```
Comment (wilsonyou) [inline docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md]: The command to start all services should be given

File: docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 4]
```diff
@@ -47,9 +47,13 @@ nebula> GET CONFIGS graph:load_data_interval_secs
...
+
+## 同一台主机先后用于单机测试和集群测试，storaged 服务无法正常启动（终端上显示的 storaged 服务的监听端口是红色的）
+
+查看 storged 服务的日志，若发现 "wrong cluster" 的报错信息，则可能的出错原因是单机测试和集群测试时的 Nebula Graph 生成的 cluster id 不一致，需要删除 cluster.id 文件和 data 目录后，重启服务。
```
Comment (wilsonyou) [inline docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md]: Please provide the directory where logs are located

File: docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 5]
```diff
@@ -45,14 +45,18 @@ nebula> GET CONFIGS graph:load_data_interval_secs
...
 
-## 使用 docker 启动后，执行命令时报错
+## 使用 Docker 启动后，执行命令时报错
 
-可能的原因是 docker 的 IP 地址和默认配置中的监听地址不一致(默认是 172.17.0.2)，因此这里需要修改默认配置中的监听地址.
+可能的原因是 Docker 的 IP 地址和默认配置中的监听地址不一致(默认是 172.17.0.2)，因此这里需要修改默认配置中的监听地址。
 
-1. 首先在容器中执行 ifconfig 命令，查看您的容器地址，这里假设您的容器地址是172.17.0.3，那么就意味着您需要修改默认配置的IP地址。
+1. 首先在容器中执行 `ifconfig` 命令，查看您的容器地址，这里假设您的容器地址是172.17.0.3，那么就意味着您需要修改默认配置的IP地址。
 2. 然后进入配置目录(cd /usr/local/nebula/etc), 查找所有IP地址配置的位置(grep "172.17.0.2" . -r)。
```
Comment (whitewum) [inline docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md]: `grep "172.17.0.2" . -r`

File: docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 6]
```diff
@@ -45,14 +45,18 @@ nebula> GET CONFIGS graph:load_data_interval_secs
...
 3. 修改上一步查到的所有IP地址为您的容器地址(172.17.0.3)。
-4. 最后重新启动所有服务。
+4. 最后重新启动所有服务(/usr/local/nebula/scripts/nebula.service start all)。
```
Comment (whitewum) [inline docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md]: `/usr/local/nebula/scripts/nebula.service start all`

**Pull request merged**